docs/mobility/slam/_index.md

```diff
@@ -19,6 +19,13 @@ Stability is not guaranteed.
 Breaking changes are likely to occur, and occur often.
 {{% /alert %}}
 
+{{% alert title="Temporarily not available in live mode" color="caution" %}}
+
+Running models of the SLAM service in live mode is temporarily disabled.
```

Comment:
similar suggestion for mentioning cloudslam here

Reply:
ah, it might be a bit confusing to just say cloudslam bc we don't refer to it that way in the docs atm-- going to specify in live and also in the cloud, does that make sense?

Reply:
yeah I think that should work